guides/common/modules/proc_exporting-a-repository-incrementally-in-a-syncable-format.adoc

```diff
@@ -0,0 +1,28 @@
+[id="Exporting_Repository_Incrementally_in_a_Syncable_Format_{context}"]
+= Exporting a Repository Incrementally in a Syncable Format
+
+Exporting a repository can be a very costly operation in terms of system resources.
+A typical {RHEL} tree may occupy several gigabytes of space on {ProjectServer}.
```

Comment:
This is missing the "orcharhino only" sentence from above.

Reply:
@maximiliankolb Which sentence is that?

Reply:
````
ifdef::orcharhino[]
The size of the exported Content View versions depends on the number of products.
endif::[]
````

See guides/common/modules/proc_exporting-a-content-view-version-incrementally.adoc in your PR.